### ISSUE TITLE
OpenBLAS Warning on ARM: "Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option."

### ISSUE THREAD

**Issue by Kenshiro-28** [ARM-software/Tool-Solutions#214]
I get this error when running torch on ARM on Docker image python:3.11-slim:

OpenBLAS Warning : Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option.

When running the same code on x86 and the same Docker image it works without errors:

This is the relevant code:

import torch
from diffusers import DiffusionPipeline, KDPM2DiscreteScheduler

...

pipe = DiffusionPipeline.from_pretrained(model, custom_pipeline = "lpw_stable_diffusion", torch_dtype = torch.float32, safety_checker = dummy_checker)
pipe.scheduler = KDPM2DiscreteScheduler.from_config(pipe.scheduler.config, use_karras_sigmas = True, algorithm_type="sde-dpmsolver++")

...

image = pipe.text2img(prompt, negative_prompt = negative_prompt, width = IMAGE_WIDTH, height = IMAGE_HEIGHT, max_embeddings_multiples = 3, num_inference_steps = INFERENCE_STEPS).images[0]

**Comment by Kenshiro-28** [comment]
Opened by error.
